docs/topics/native/native-objc-interop.md

```diff
@@ -16,6 +16,25 @@ Kotlin modules can be used in Swift/Objective-C code if compiled into a
 framework ([see here for how to declare binaries](multiplatform-build-native-binaries.md#declare-binaries)).
 See [Kotlin Multiplatform Mobile Sample](https://github.com/Kotlin/kmm-basic-sample) for an example.
 
+### Hiding Kotlin declarations
+
+You can choose not to export Kotlin declaration to Objective-C and Swift with special annotations:
+
+* `@HiddenFromObjC` hides a Kotlin declaration from Objective-C and, consequently, Swift. The annotation instructs the
+  Kotlin compiler not to export a function or property to Objective-C, making your Kotlin code more Objective-C/Swift-friendly.
```

Comment:
Maybe we should rephrase it to "The annotation disables a function or prop export to objc" ? 

WDYT?